components/store/StoreCard.js

```diff
@@ -57,44 +51,37 @@ export default function StoreCard({
   return (
     <TouchableOpacity
```

Comment:
This should be conditional on if it's from the `StoreList`. Currently, it seems to be broken bc clicking does nothing but it's navigating to itself